src/sentry/monitors/models.py

```diff
@@ -146,7 +150,7 @@ class CheckInStatus:
     status was reported by the monitor itself (was not synthetic)
     """
 
-    SYNTHETIC_TERMINAL_VALUES = (MISSED, TIMEOUT)
+    SYNTHETIC_TERMINAL_VALUES = (MISSED, TIMEOUT, UNKNOWN)
     """
     Values indicating the montior is in a terminal "synthetic" status. These
```

Comment:
is this comment still accurate? weren't you saying UNKNOWNS could be later on marked as OK?

Reply:
Let me update to noe that it may move out of this state